HelpSource/Reference/SCDocSyntax.schelp

```diff
@@ -413,7 +423,8 @@ optsubsubsections ::= [ subsubsections ]
 subsubsections    ::= subsubsections subsubsection
                     | subsubsection
                     | body
-subsubsection     ::= "METHOD::" methnames optMETHODARGS eol methodbody
+subsubsection     ::= "SUBSUBSECTION::" words2 eol optbody
```

Comment:
Will there be any conflicts between methods and subsubsections?

Reply:
I do not think there is any conflict between methods and subsubsections.
Please see the following errors are all the errors and warnings when evaluating `SCDoc.renderAll`:
```
SCDoc: Rendering all documents
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/ServerOptionsGui/HelpSource/Classes/ServerOptionsGui.schelp
  Broken link: 'Server'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/ServerOptionsGui/HelpSource/Classes/ServerOptionsGui.schelp
  Broken link: 'ServerOptions'
WARNING: SCDoc: In /Users/prko/Downloads/supercollider/build/Install/SuperCollider/SuperCollider.app/Contents/Resources/HelpSource/Classes/Window.schelp
  Method *new has 6 args, but doc has 5 argument:: tags.
WARNING: SCDoc: In /Users/prko/Downloads/supercollider/build/Install/SuperCollider/SuperCollider.app/Contents/Resources/HelpSource/Classes/Window.schelp
  Method *new has arg named 'deprecatedServerArgument', but doc has 'argument:: scroll'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSampler/HelpSource/Classes/RedSampler.schelp
  Broken link: 'Overviews/redSampler'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/ScaleSpec.schelp
  Broken link: 'Classes/CXScale'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/BBCut/HelpSource/Classes/SQPusher2.schelp
  Method *new has 5 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/BBCut/HelpSource/Classes/SQPusher2.schelp
  Method *new has arg named 'quant', but doc has 'argument:: bpsd'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSys/HelpSource/Classes/RedLFSR4BitStream.schelp
  Broken link: 'Classes/RedNoise'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Log/HelpSource/Classes/LogWindow.schelp
  Method *new has arg named '... settings', but doc has 'argument:: logArray'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSys/HelpSource/Classes/RedLFSR4.schelp
  Broken link: 'Classes/RedNoise'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/Patch.schelp
  Broken link: 'Classes/Mx'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/PV_MagSmooth.schelp
  Broken link: 'PV_MagSmear'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/PV_MagSmooth.schelp
  Method *new has arg named 'buffer', but doc has 'argument:: chain'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/ConcatUGens/HelpSource/Classes/Concat.schelp
  Method *ar has 12 args, but doc has 11 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/GamelanPatterns/HelpSource/Classes/StringConverter.schelp
  Broken link: 'Reference/VGNotationStrings'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/VectorSpace/HelpSource/Classes/RealVector.schelp
  Method *rand has 3 args, but doc has 2 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/VectorSpace/HelpSource/Classes/RealVector.schelp
  Method *rand has arg named 'lo', but doc has 'argument:: lo, hi'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/VectorSpace/HelpSource/Classes/RealVector.schelp
  Method -zero not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkAudio.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/LPCVals.schelp
  Broken link: 'Classes/LPCAna'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/LPCVals.schelp
  Broken link: 'Classes/LPCFile'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/LPCVals.schelp
  Broken link: 'Classes/LPCFile'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/LPCVals.schelp
  Broken link: 'Classes/LPCFile'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Utopia/HelpSource/Classes/PeerGroup.schelp
  Method -sendMsg has arg named '... args', but doc has 'argument::  ... args'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Utopia/HelpSource/Classes/PeerGroup.schelp
  Method -sendBundle has arg named '... args', but doc has 'argument::  ... args'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSys/HelpSource/Classes/RedInterpolator.schelp
  Broken link: 'Overviews/EaseOverview##Ease'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/BandSplitter/HelpSource/Classes/BandSplitter2.schelp
  Method *ar has arg named 'freq', but doc has 'argument:: f1'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/AbstractPlayer.schelp
  Broken link: 'Classes/Mx'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/AbstractPlayer.schelp
  Method -busReport not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/AbstractPlayer.schelp
  Method -asKAsset not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/AbstractPlayer.schelp
  Method -swapperApp not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/AbstractPlayer.schelp
  Method -wigglyness not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SuperSampler/HelpSource/Classes/SamplerDB.schelp
  Broken link: 'Classes/Sampler'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SuperSampler/HelpSource/Classes/SamplerDB.schelp
  Method -playEnv has arg named 'numSampler', but doc has 'argument:: numSmpler'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DL.schelp
  Method *ar has 8 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DL.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DL.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: a, b, c, d'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DL.schelp
  Method *ar has arg named 'a', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DL.schelp
  Method *ar has arg named 'b', but doc has 'argument:: y0'.
ERROR: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/ddwMIDI/HelpSource/Classes/BasicMIDIControl.schelp:
  At line 63: syntax error, unexpected end of file

WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/ddwChucklib/HelpSource/Classes/BP.schelp
  Broken link: 'Tutorials/A-Practical-Guide/PG_06g_Data_Sharing.html#Communicating values between separate event patterns'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/PV_Compander.schelp
  Method *new has 4 args, but doc has 3 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/PV_Compander.schelp
  Method *new has arg named 'slopeBelow', but doc has 'argument:: slopeBelow, slopeAbove'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/sInstruments/HelpSource/Classes/SPlayFileView.schelp
  Broken link: 'Classes/SplayFile'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/Instr.schelp
  Method -makeSpecs has arg named 'argspecs', but doc has 'argument:: argSpecs'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/PV_Cutoff.schelp
  Method *new has 3 args, but doc has 2 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/PV_Cutoff.schelp
  Method *new has arg named 'bufferA', but doc has 'argument:: buffer'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/PV_Cutoff.schelp
  Method *new has arg named 'bufferB', but doc has 'argument:: wipe'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/greenSys/HelpSource/Classes/GreenTest.schelp
  Broken link: 'Classes/ReplaceBadValues'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/MusicalNotation/HelpSource/Classes/Chord.schelp
  Method *at has arg named 'key', but doc has 'argument:: input'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SimpleDMXControl/HelpSource/Classes/SimpleDMX.schelp
  Method -sendDMX has 2 args, but doc has 1 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Utopia/HelpSource/Classes/AddrBook.schelp
  Method -send has arg named '... msg', but doc has 'argument::  ... msg'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Utopia/HelpSource/Classes/AddrBook.schelp
  Method -sendAll has arg named '... msg', but doc has 'argument::  ... msg'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Utopia/HelpSource/Classes/AddrBook.schelp
  Method -sendAllBundle has arg named '... msg', but doc has 'argument::  ... msg'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Utopia/HelpSource/Classes/AddrBook.schelp
  Method -sendExcluding has arg named '... msg', but doc has 'argument::  ... msg'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Automation/HelpSource/Classes/AutomationGui.schelp
  Broken link: 'Tutorials/Automation##Automation Tutorial'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/autopan/HelpSource/Classes/AutoPan.schelp
  Method *new has arg named 'maxSize', but doc has 'argument:: arga'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/autopan/HelpSource/Classes/AutoPan.schelp
  Method -init not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/PortedPlugins/HelpSource/Classes/PhasorModal.schelp
  Broken link: 'TODO'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/PortedPlugins/HelpSource/Classes/PhasorModal.schelp
  Method *ar has 6 args, but doc has 2 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/PortedPlugins/HelpSource/Classes/PhasorModal.schelp
  Method *ar has arg named 'input', but doc has 'argument:: TODO'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/PortedPlugins/HelpSource/Classes/PhasorModal.schelp
  Method *ar has arg named 'freq', but doc has 'argument:: TODO'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DC.schelp
  Method *ar has 9 args, but doc has 8 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DC.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DC.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: rateu'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DC.schelp
  Method *ar has arg named 'urate', but doc has 'argument:: ratew'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DC.schelp
  Method *ar has arg named 'wrate', but doc has 'argument:: b0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DC.schelp
  Method *ar has arg named 'b0', but doc has 'argument:: b1'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DC.schelp
  Method *ar has arg named 'b1', but doc has 'argument:: i'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DC.schelp
  Method *ar has arg named 'i', but doc has 'argument:: initu'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DC.schelp
  Method *ar has arg named 'u0', but doc has 'argument:: initw'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkControl.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DN.schelp
  Method *ar has 9 args, but doc has 7 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DN.schelp
  Method *ar has arg named 's', but doc has 'argument:: s, r, b'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DN.schelp
  Method *ar has arg named 'r', but doc has 'argument:: h'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DN.schelp
  Method *ar has arg named 'b', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DN.schelp
  Method *ar has arg named 'h', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DN.schelp
  Method *ar has arg named 'x0', but doc has 'argument:: z0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/SLUGens/HelpSource/Classes/NL2.schelp
  Method *ar has arg named 'maxsizea', but doc has 'argument:: maxasize'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/SLUGens/HelpSource/Classes/NL2.schelp
  Method *ar has arg named 'maxsizeb', but doc has 'argument:: maxbsize'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/StreamKrDur.schelp
  Method -wigglyness not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DL.schelp
  Method *ar has 9 args, but doc has 8 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DL.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DL.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: rateu'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DL.schelp
  Method *ar has arg named 'urate', but doc has 'argument:: ratew'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DL.schelp
  Method *ar has arg named 'wrate', but doc has 'argument:: b0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DL.schelp
  Method *ar has arg named 'b0', but doc has 'argument:: b1'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DL.schelp
  Method *ar has arg named 'b1', but doc has 'argument:: i'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DL.schelp
  Method *ar has arg named 'i', but doc has 'argument:: initu'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DL.schelp
  Method *ar has arg named 'u0', but doc has 'argument:: initw'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/say/HelpSource/Say.schelp
  Broken link: 'Classes/Speech'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DN.schelp
  Method *ar has 8 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DN.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DN.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: a, b, c, d'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DN.schelp
  Method *ar has arg named 'a', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DN.schelp
  Method *ar has arg named 'b', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/greenSys/HelpSource/Classes/GreenNoise.schelp
  Broken link: 'Classes/RedNoise'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/GUICV/HelpSource/Overviews/GUICVOverview.schelp
  Broken link: 'Overviews/GFX_Overview'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/AllBuf/HelpSource/Classes/AllBuf.schelp
  Method *new has 4 args, but doc has 3 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/AllBuf/HelpSource/Classes/AllBuf.schelp
  Method *new has arg named 'force', but doc has 'argument:: verbose'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/AllBuf/HelpSource/Classes/AllBuf.schelp
  Method -def not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/Sample.schelp
  Method *roster not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/Sample.schelp
  Method -rand not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/RosslerL.schelp
  Method *ar has 8 args, but doc has 7 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/RosslerL.schelp
  Method *ar has arg named 'freq', but doc has 'argument:: in'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/RosslerL.schelp
  Method *ar has arg named 'a', but doc has 'argument:: freq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/RosslerL.schelp
  Method *ar has arg named 'b', but doc has 'argument:: a, b, c'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/RosslerL.schelp
  Method *ar has arg named 'c', but doc has 'argument:: h'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/RosslerL.schelp
  Method *ar has arg named 'h', but doc has 'argument:: xi'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/RosslerL.schelp
  Method *ar has arg named 'xi', but doc has 'argument:: yi'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/RosslerL.schelp
  Method *ar has arg named 'yi', but doc has 'argument:: zi'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Notator/HelpSource/Classes/SPN.schelp
  Method *convert not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Notator/HelpSource/Classes/SPN.schelp
  Method *midioct not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/splines/HelpSource/Classes/BSplineGui.schelp
  Broken link: 'Classes/SplineGui.'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/BBCut/HelpSource/Classes/MotifCutProc.schelp
  Method -initBBCutProc11 not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/scparco/HelpSource/Classes/ScpParserState.schelp
  Method *new has 6 args, but doc has 5 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/scparco/HelpSource/Classes/ScpParserState.schelp
  Method -init has 6 args, but doc has 5 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Modality-toolkit/Modality/HelpSource/Classes/MHexPad.schelp
  Broken link: 'Overviews/MOdality'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSampler/HelpSource/Classes/RedDiskInSampler.schelp
  Broken link: 'Overviews/redSampler'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BlackrainUGens/HelpSource/Classes/AmplitudeMod.schelp
  Method *ar has arg named 'in', but doc has 'argument:: input'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/JITLibExtensions/HelpSource/Classes/NPVoicerL.schelp
  Grouped methods [prime, put, release, releaseAll] do not have the same argument signature.
WARNING: SCDoc: In /Users/prko/Downloads/supercollider/build/Install/SuperCollider/SuperCollider.app/Contents/Resources/HelpSource/Classes/Signal.schelp
  Method *hanningWindow has 3 args, but doc has 2 argument:: tags.
WARNING: SCDoc: In /Users/prko/Downloads/supercollider/build/Install/SuperCollider/SuperCollider.app/Contents/Resources/HelpSource/Classes/Signal.schelp
  Method *hammingWindow has 3 args, but doc has 2 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DC.schelp
  Method *ar has 6 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DC.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DC.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: a, b'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DC.schelp
  Method *ar has arg named 'a', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DC.schelp
  Method *ar has arg named 'b', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/TuningLib/HelpSource/Classes/Key.schelp
  Broken link: 'Classes/ScaleInfo'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/PVSynth.schelp
  Broken link: 'Classes/PVFile'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/ddwPlug/HelpSource/Classes/Plug.schelp
  Broken link: 'Guides/ServerTiming.html'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/ddwPlug/HelpSource/Classes/Plug.schelp
  Broken link: 'Classes/Syn.html#Three-point%20modulation'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/ddwPlug/HelpSource/Classes/Plug.schelp
  Broken link: 'Classes/Syn.html#Three-point%20modulation'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/ddwPlug/HelpSource/Classes/Plug.schelp
  Broken link: 'Classes/Syn.html#Three-point%20modulation'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/mitola/HelpSource/Classes/MtlScalaCalculator.schelp
  Method *prime_factors not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DN.schelp
  Method *ar has 4 args, but doc has 3 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DN.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DN.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DN.schelp
  Method *ar has arg named 'x0', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/greenSys/HelpSource/Classes/GreenFreqScope.schelp
  Method *new has arg named 'maxSize', but doc has 'argument:: parent'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkScore.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Guides/Patch-spawning-and-patterns.schelp
  Broken link: 'Classes/InstrEventListPlayer'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkObj.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/AbstractPlayerGui.schelp
  Method -enableMIDI not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Fomus/HelpSource/Classes/Fomus.schelp
  Broken link: 'Event Stream'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Fomus/HelpSource/Classes/Fomus.schelp
  Method -add has arg named 'that', but doc has 'argument:: stuffIn'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSampler/HelpSource/Classes/RedLoop.schelp
  Broken link: 'Overviews/redSampler'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/PV_PlayBuf.schelp
  Method *new has arg named 'loop', but doc has 'argument:: binStart'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Singleton/HelpSource/Classes/Singleton.schelp
  Method *new has arg named '... settings', but doc has 'argument:: ...settings'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/DistortionUGens/HelpSource/Classes/SineShaper.schelp
  Broken link: 'Classes/SmoothingDecimator'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSampler/HelpSource/Classes/RedSamp.schelp
  Broken link: 'Overviews/redSampler'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/KMeansRT.schelp
  Broken link: 'Classes/KMeans'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkNoteObject.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/magnituderesponseview/HelpSource/Classes/FOSKernel.schelp
  Method *filterKind not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SimpleDMXControl/HelpSource/Classes/SimpleEnttecDMXProMk2.schelp
  Method *new has arg named 'universeSizes', but doc has 'argument:: universeSize'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/SLUGens/HelpSource/Classes/PrintVal.schelp
  Method *ar not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSampler/HelpSource/Classes/RedDiskInSamplerGiga.schelp
  Broken link: 'Overviews/redSampler'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/GamelanPatterns/HelpSource/Classes/BalunganReader.schelp
  Broken link: 'Reference/VGNotationStrings'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/GamelanPatterns/HelpSource/Classes/BalunganReader.schelp
  Broken link: 'Reference/VGNotationStrings'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/KeyPlayer/HelpSource/KeyPlayer.schelp
  Broken link: '../KeyPlayerExample.scd'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkGroup.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Downloads/supercollider/build/Install/SuperCollider/SuperCollider.app/Contents/Resources/HelpSource/Classes/Document.schelp
  Method -save has arg named 'docPath', but doc has 'argument:: path'.
WARNING: SCDoc: In /Users/prko/Downloads/supercollider/build/Install/SuperCollider/SuperCollider.app/Contents/Resources/HelpSource/Classes/UGen.schelp
  Method -thresh2 has arg named 'function', but doc has 'argument:: thresh'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/GamelanSynthesis/HelpSource/Classes/VGSound.schelp
  Broken link: 'Guides/VGStartup'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Utopia/HelpSource/Classes/OSCObjectSpace.schelp
  Broken link: 'Classes/IdentityDictionary etc.'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SuperSampler/HelpSource/Classes/SampleDescript.schelp
  Broken link: 'Classes/SCMIR'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SuperSampler/HelpSource/Classes/SampleDescript.schelp
  Method *new has 13 args, but doc has 12 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SuperSampler/HelpSource/Classes/SampleDescript.schelp
  Method *new has arg named 'normalize', but doc has 'argument:: server'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SuperSampler/HelpSource/Classes/SampleDescript.schelp
  Method *new has arg named 'server', but doc has 'argument:: action'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SuperSampler/HelpSource/Classes/SampleDescript.schelp
  Method -play has 6 args, but doc has 3 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SuperSampler/HelpSource/Classes/SampleDescript.schelp
  Method -play has arg named 'out', but doc has 'argument:: outbus'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/SuperSampler/HelpSource/Classes/SampleDescript.schelp
  Method -activeDissonanceData not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/PV_SpectralMap.schelp
  Broken link: 'PV_SpectralEnhance'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DN.schelp
  Method *ar has 5 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DN.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DN.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: k'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DN.schelp
  Method *ar has arg named 'k', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DN.schelp
  Method *ar has arg named 'x0', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/lfsaw.de/HelpSource/Classes/PhasorTJ.schelp
  Method *ar has 7 args, but doc has 6 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/lfsaw.de/HelpSource/Classes/PhasorTJ.schelp
  Method *ar has arg named 'offset', but doc has 'argument:: jitter'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/lfsaw.de/HelpSource/Classes/PhasorTJ.schelp
  Method *ar has arg named 'jitter', but doc has 'argument:: jitterFreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/lfsaw.de/HelpSource/Classes/PhasorTJ.schelp
  Method *ar has arg named 'jitterFreq', but doc has 'argument:: jitterType'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/lfsaw.de/HelpSource/Classes/PhasorTJ.schelp
  Method *ar has arg named 'jitterType', but doc has 'argument:: phaseType'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/HasSubject.schelp
  Method -state not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/VectorSpace/HelpSource/Classes/AbstractVector.schelp
  Method -clip has 2 args, but doc has 1 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Notator/HelpSource/Classes/PitchClassSet.schelp
  Method *names has 2 args, but doc has 1 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Notator/HelpSource/Classes/PitchClassSet.schelp
  Method *num has 2 args, but doc has 1 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/splines/HelpSource/Classes/SplineGen.schelp
  Broken link: 'Classes/KrSplinePlayer'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/splines/HelpSource/Classes/SplineGen.schelp
  Broken link: 'Classes/SplineFr'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/KeyPlayer/HelpSource/KeyPlayerGui.schelp
  Broken link: 'Reference/KeyLoop'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkNote.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/DEINDUGens/HelpSource/Classes/Greyhole.schelp
  Method *ar has 8 args, but doc has 7 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/DEINDUGens/HelpSource/Classes/Greyhole.schelp
  Method *ar has arg named 'size', but doc has 'argument:: diff'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/DEINDUGens/HelpSource/Classes/Greyhole.schelp
  Method *ar has arg named 'diff', but doc has 'argument:: feedback'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/DEINDUGens/HelpSource/Classes/Greyhole.schelp
  Method *ar has arg named 'feedback', but doc has 'argument:: modDepth'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/DEINDUGens/HelpSource/Classes/Greyhole.schelp
  Method *ar has arg named 'modDepth', but doc has 'argument:: modFreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/SLUGens/HelpSource/Classes/Max.schelp
  Method *ar not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkPGroup.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/TagSystemUGens/HelpSource/Classes/DbufTag.schelp
  Method *new has 6 args, but doc has 1 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/PortedPlugins/HelpSource/Classes/AnalogFoldOsc.schelp
  Broken link: 'HelpSource/Classes/LPG'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Modality-toolkit/Modality/HelpSource/Tutorials/ModalityTutorial.schelp
  Broken link: 'Classes/MFunc'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/ddwCommon/HelpSource/Classes/GlobalControlMIDIOut.schelp
  Method *new has arg named 'midisender', but doc has 'argument:: midiout'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/JITLibExtensions/HelpSource/Classes/BufEnvir.schelp
  Grouped methods [read, cue] do not have the same argument signature.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/JITLibExtensions/HelpSource/Classes/BufEnvir.schelp
  Grouped methods [readAll, cueAll] do not have the same argument signature.
WARNING: SCDoc: In /Users/prko/Downloads/supercollider/build/Install/SuperCollider/SuperCollider.app/Contents/Resources/HelpSource/Classes/MultiOutUGen.schelp
  Method -initOutputs has 2 args, but doc has 1 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/LPCSynth.schelp
  Broken link: 'Classes/LPCAna'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/LPCSynth.schelp
  Broken link: 'Classes/LPCFile'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/LPCSynth.schelp
  Broken link: 'Classes/LPCFile'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkBuffer.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkSynthDef.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/FFTCrest.schelp
  Method *kr has arg named 'buffer', but doc has 'argument:: chain'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DL.schelp
  Method *ar has 5 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DL.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DL.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: k'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DL.schelp
  Method *ar has arg named 'k', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DL.schelp
  Method *ar has arg named 'x0', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/that/HelpSource/Classes/That.schelp
  Method *new has 5 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DC.schelp
  Method *ar has 5 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DC.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DC.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: k'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DC.schelp
  Method *ar has arg named 'k', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Standard2DC.schelp
  Method *ar has arg named 'x0', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Dimple/HelpSource/Classes/InteractiveEnvironment.schelp
  Broken link: 'Classes/AmbisonicPan'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/SCMIRUGens/HelpSource/Classes/BeatStatistics.schelp
  Method *kr has 3 args, but doc has 1 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/splines/HelpSource/Classes/LinearSpline.schelp
  Method *new has arg named 'isClosed', but doc has 'argument:: isClosed [boolean]'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/BBCut/HelpSource/Classes/AnalyseEventsDatabase.schelp
  Method *new has 4 args, but doc has 3 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/BBCut/HelpSource/Classes/AnalyseEventsDatabase.schelp
  Method *new has arg named 'numChannels', but doc has 'argument:: numChan'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/BBCut/HelpSource/Classes/AnalyseEventsDatabase.schelp
  Method -analyse has 7 args, but doc has 5 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/TabbedView2/HelpSource/Classes/TabbedView2.schelp
  Method -autoArchive has arg named '... args', but doc has 'argument:: symbols'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/TabbedView2/HelpSource/Classes/TabbedView2.schelp
  Method -saveStateToArchive has arg named '... args', but doc has 'argument:: symbols'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/TabbedView2/HelpSource/Classes/TabbedView2.schelp
  Method -restoreStateFromArchive has arg named '... args', but doc has 'argument:: symbols'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/TabbedView2/HelpSource/Classes/TabbedView2.schelp
  Method -tabHeight_ has arg named 'val', but doc has 'argument:: int'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/FreeAfter/HelpSource/Guides/FreeAfter overview.schelp
  Broken link: 'Classes/Meta_CmdPeriod'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/FMHDecode1.schelp
  Method *ar1 has arg named 'mul', but doc has 'argument:: scaleflag'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/FMHDecode1.schelp
  Method *ar1 has arg named 'add', but doc has 'argument:: mul'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/FMHDecode1.schelp
  Method *ar1 has arg named 'scaleflag', but doc has 'argument:: add'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/MusicalNotation/HelpSource/Classes/Progression.schelp
  Method *at has arg named 'key', but doc has 'argument:: input'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/TabbedView/HelpSource/Classes/TabbedView.schelp
  Broken link: 'etc.'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/TabbedView/HelpSource/Classes/TabbedView.schelp
  Method *newRGB not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/TabbedView/HelpSource/Classes/TabbedView.schelp
  Method *newRGBLabels not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/TabbedView/HelpSource/Classes/TabbedView.schelp
  Method -tabHeight_ has arg named 'val', but doc has 'argument:: int'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/splines/HelpSource/Classes/SplineGui.schelp
  Method -spec_ has arg named 'sp', but doc has 'argument:: spec'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/splines/HelpSource/Classes/SplineGui.schelp
  Method -background_ has arg named 'c', but doc has 'argument:: color'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/PV_RecordBuf.schelp
  Broken link: 'Classes/PV_Player'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/PV_RecordBuf.schelp
  Broken link: 'Classes/PV_Player'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/TBetaRand.schelp
  Method *ar has 5 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/TBetaRand.schelp
  Method *ar has arg named 'prob1', but doc has 'argument:: prob1, prob2'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/TBetaRand.schelp
  Method *ar has arg named 'prob2', but doc has 'argument:: trig'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkProtoNotes.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/PV_ExtractRepeat.schelp
  Method *new has arg named 'buffer', but doc has 'argument:: array'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/PV_ExtractRepeat.schelp
  Method *new has arg named 'loopbuf', but doc has 'argument:: chain'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/PV_ExtractRepeat.schelp
  Method *new has arg named 'loopdur', but doc has 'argument:: loopbuf'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/PV_ExtractRepeat.schelp
  Method *new has arg named 'memorytime', but doc has 'argument:: loopdur'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/PV_ExtractRepeat.schelp
  Method *new has arg named 'which', but doc has 'argument:: memorytime'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/PV_ExtractRepeat.schelp
  Method *new has arg named 'ffthop', but doc has 'argument:: which'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/MCLDUGens/HelpSource/Classes/PV_ExtractRepeat.schelp
  Method *new has arg named 'thresh', but doc has 'argument:: ffthop'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Ctk/HelpSource/Classes/CtkEvent.schelp
  Broken link: 'Overviews/CTK'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Morse/HelpSource/Classes/Morse.schelp
  Broken link: 'MorseDict'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DN.schelp
  Method *ar has 9 args, but doc has 8 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DN.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DN.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: rateu'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DN.schelp
  Method *ar has arg named 'urate', but doc has 'argument:: ratew'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DN.schelp
  Method *ar has arg named 'wrate', but doc has 'argument:: b0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DN.schelp
  Method *ar has arg named 'b0', but doc has 'argument:: b1'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DN.schelp
  Method *ar has arg named 'b1', but doc has 'argument:: i'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DN.schelp
  Method *ar has arg named 'i', but doc has 'argument:: initu'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Fhn2DN.schelp
  Method *ar has arg named 'u0', but doc has 'argument:: initw'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BlackrainUGens/HelpSource/Classes/BMoog.schelp
  Method *ar has 5 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DL.schelp
  Method *ar has 9 args, but doc has 7 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DL.schelp
  Method *ar has arg named 's', but doc has 'argument:: s, r, b'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DL.schelp
  Method *ar has arg named 'r', but doc has 'argument:: h'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DL.schelp
  Method *ar has arg named 'b', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DL.schelp
  Method *ar has arg named 'h', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DL.schelp
  Method *ar has arg named 'x0', but doc has 'argument:: z0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/Interface.schelp
  Broken link: 'Classes/Mx'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/SLUGens/HelpSource/Classes/WaveTerrain.schelp
  Method *ar has 5 args, but doc has 3 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/scparco/HelpSource/Classes/ScpParser.schelp
  Method -log has 2 args, but doc has 1 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/SLUGens/HelpSource/Classes/WeaklyNonlinear.schelp
  Method *ar has 11 args, but doc has 7 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/sInstruments/HelpSource/Classes/SPlayFileDiskView.schelp
  Broken link: 'Classes/SplayFileDisk'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/HyperDisCo/HelpSource/OSCRouterClient.schelp
  Broken link: 'Reference/OpenSoundControl'
WARNING: SCDoc: In /Users/prko/Downloads/supercollider/build/Install/SuperCollider/SuperCollider.app/Contents/Resources/HelpSource/Classes/SequenceableCollection.schelp
  Method -adjustZeroXs has arg named 'zeroXBufs', but doc has 'argument:: zeroXBuf'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/KtlLoop/HelpSource/EventLoop.schelp
  Broken link: 'Classes/GamePad'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DC.schelp
  Method *ar has 8 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DC.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DC.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: a, b, c, d'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DC.schelp
  Method *ar has arg named 'a', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Latoocarfian2DC.schelp
  Method *ar has arg named 'b', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/KtlLoop/HelpSource/KeyLoop.schelp
  Broken link: 'Reference/KeyPlayerGui'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DN.schelp
  Method *ar has 6 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DN.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DN.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: a, b'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DN.schelp
  Method *ar has arg named 'a', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DN.schelp
  Method *ar has arg named 'b', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DC.schelp
  Method *ar has 4 args, but doc has 3 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DC.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DC.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DC.schelp
  Method *ar has arg named 'x0', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSys/HelpSource/Classes/RedTween.schelp
  Broken link: 'Overviews/EaseOverview##Ease'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSys/HelpSource/Classes/RedTween.schelp
  Broken link: 'Overviews/EaseOverview##Ease'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/GamelanPatterns/HelpSource/Classes/EventScoreReader.schelp
  Broken link: 'Reference/VGNotationStrings'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/Editor.schelp
  Method -state not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/SLUGens/HelpSource/Classes/WeaklyNonlinear2.schelp
  Method *ar has 11 args, but doc has 7 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/GamelanPatterns/HelpSource/Classes/KendhanganReader.schelp
  Broken link: 'Reference/VGNotationStrings'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DC.schelp
  Method *ar has 9 args, but doc has 7 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DC.schelp
  Method *ar has arg named 's', but doc has 'argument:: s, r, b'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DC.schelp
  Method *ar has arg named 'r', but doc has 'argument:: h'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DC.schelp
  Method *ar has arg named 'b', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DC.schelp
  Method *ar has arg named 'h', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Lorenz2DC.schelp
  Method *ar has arg named 'x0', but doc has 'argument:: z0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/mitola/HelpSource/Classes/MtlMitola.schelp
  Method *new has 14 args, but doc has 13 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/mitola/HelpSource/Classes/MtlMitola.schelp
  Method -init has 14 args, but doc has 13 argument:: tags.
ERROR: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/splines/HelpSource/Classes/SplineMapper.schelp:
  At line 150: syntax error, unexpected end of file

WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSampler/HelpSource/Classes/RedLoopDisk.schelp
  Broken link: 'Overviews/redSampler'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/redSampler/HelpSource/Classes/RedDiskInPlayer.schelp
  Broken link: 'Overviews/redSampler'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/PortedPlugins/HelpSource/Classes/LockhartWavefolder.schelp
  Method *ar has 2 args, but doc has 1 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/crucial-library/HelpSource/Classes/KrPlayer.schelp
  Method -wigglyness not found.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/JoshUGens/HelpSource/Classes/PVInfo.schelp
  Broken link: 'Classes/PVFile'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DL.schelp
  Method *ar has 6 args, but doc has 4 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DL.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DL.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: a, b'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DL.schelp
  Method *ar has arg named 'a', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Henon2DL.schelp
  Method *ar has arg named 'b', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/Utopia/HelpSource/Classes/SynthDescRelay.schelp
  Broken link: 'etc.'
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DL.schelp
  Method *ar has 4 args, but doc has 3 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DL.schelp
  Method *ar has arg named 'minfreq', but doc has 'argument:: minfreq, maxfreq'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DL.schelp
  Method *ar has arg named 'maxfreq', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/Gbman2DL.schelp
  Method *ar has arg named 'x0', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/LorenzTrig.schelp
  Method *ar has 9 args, but doc has 7 argument:: tags.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/LorenzTrig.schelp
  Method *ar has arg named 's', but doc has 'argument:: s, r, b'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/LorenzTrig.schelp
  Method *ar has arg named 'r', but doc has 'argument:: h'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/LorenzTrig.schelp
  Method *ar has arg named 'b', but doc has 'argument:: x0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/LorenzTrig.schelp
  Method *ar has arg named 'h', but doc has 'argument:: y0'.
WARNING: SCDoc: In /Users/prko/Library/Application Support/SuperCollider/Extensions/SC3plugins/BhobUGens/HelpSource/Classes/LorenzTrig.schelp
  Method *ar has arg named 'x0', but doc has 'argument:: z0'.
SCDoc: Done!
-> SCDoc
```